addons/udes_mrp/__manifest__.py

```diff
@@ -19,13 +19,14 @@
     'depends': [
         'base',
         'mrp',
+        'stock',
```

Comment:
I wouldn't consider this necessary since the `mrp` module already depends on `stock`. since `udes_mrp` is now directly modifying `stock` things (`stock.picking.type`) it's OK to leave it as explicit now.

Reply:
I went and removed it but it seems I haven't understood the latter sentence - should I leave it or remove it? :P

Reply:
I communicated poorly, and don't really mind, but I'd prefer not having it. Remove it.